Bottom: 87232eed4299ac03f5e1ad85fbd4debecb54a612
Top:    e58ac8dab3af7e0858f0d7dec9e76eacda8b88fc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 01:57:49 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 16247a8..385351e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
